idea/src/org/jetbrains/kotlin/idea/KotlinFoldingBuilder.kt

```diff
@@ -97,12 +97,28 @@ class KotlinFoldingBuilder : CustomFoldingBuilder(), DumbAware {
     }
 
     override fun getLanguagePlaceholderText(node: ASTNode, range: TextRange): String = when {
-        node.elementType == KtTokens.BLOCK_COMMENT -> "/.../"
-        node.elementType == KDocTokens.KDOC -> "/**...*/"
+        node.elementType == KtTokens.BLOCK_COMMENT -> "/${getFirstLineOfComment(node)}.../"
+        node.elementType == KDocTokens.KDOC -> "/**${getFirstLineOfComment(node)}...*/"
         node.psi is KtImportList -> "..."
         else ->  "{...}"
     }
 
+    private fun getFirstLineOfComment(node: ASTNode): String {
+        val targetCommentLine = node.text.split("\n").firstOrNull {
+            getCommentContents(it).isNotEmpty()
+        } ?: return ""
+        return getCommentContents(targetCommentLine)
+    }
+
+    private fun getCommentContents(line: String): String {
+        val comment = KotlinCommenter()
+        return line.trim()
+                .replace(comment.documentationCommentPrefix!!, "")
+                .replace(comment.blockCommentPrefix!!, "")
+                .replace(comment.blockCommentSuffix!!, "")
+                .replace(comment.documentationCommentLinePrefix!!, "")
```

Comment:
Please make sure that the line has at most one leading and at most one trailing space after you've remove the prefix and suffix